File: dto-mapping-benchmark/src/main/java/com/example/dtomappingbenchmark/v3/repository/UserRepository.java
package com.example.dtomappingbenchmark.v3.repository;

import com.example.dtomappingbenchmark.v3.entity.User;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository("userRepositoryV3")
public interface UserRepository extends JpaRepository<User, Long> {
}

File: object-mapping-benchmark-v1/src/main/kotlin/yu/selab/cj/objectmappingbenchmarkv1/user/repository/UserRepository.kt
package yu.selab.cj.objectmappingbenchmarkv1.user.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import yu.selab.cj.objectmappingbenchmarkv1.user.entity.User

@Repository
interface UserRepository : JpaRepository<User, Long> {
}